src/main/haskell/language-kore/src/Data/Kore/ASTPrettyPrint.hs

```diff
@@ -9,7 +9,7 @@ import           Control.Monad.Writer
 
 import           Data.Kore.AST
 import           Data.Kore.Parser.CString (escapeCString)
-import           Data.List (intersperse)
+import           Data.List                (intersperse)
 
 {-# ANN module "HLint: ignore Use record patterns" #-}
```

Comment:
Why not use a record pattern match? I think would have the same safety effect.

Reply:
I think it doesn't, and I thought that the comment below explains it, please correct me if I am wrong. Consider the following case:
```
prettyPrint p@(And _ _ _) =
   stuff (andSort p) (andFirst p) (andSecond p)
```
if I add a field to 'And', then I'll get a compilation error for the `(And _ _ _)` part. However, the following code:
```
prettyPrint And {andSort=as, andFirst=af, andSecond ad} =
   stuff as af ad
```
will happily compile and run in the same case. This means that one may add a field and forget to update the pretty-printer.